Assets/MRTK/Providers/Oculus/XRSDK/MRTK.ThirdParty/MRTK-Quest-Utils/Scripts/Debug/HandJointFollow.cs

```diff
@@ -0,0 +1,79 @@
+﻿//------------------------------------------------------------------------------ -
+//MRTK - Quest
+//https ://github.com/provencher/MRTK-Quest
+//------------------------------------------------------------------------------ -
+//
+//MIT License
+//
+//Copyright(c) 2020 Eric Provencher
+//
+//Permission is hereby granted, free of charge, to any person obtaining a copy
+//of this software and associated documentation files(the "Software"), to deal
+//in the Software without restriction, including without limitation the rights
+//to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//copies of the Software, and to permit persons to whom the Software is
+//furnished to do so, subject to the following conditions :
+//
+//The above copyright notice and this permission notice shall be included in all
+//copies or substantial portions of the Software.
+//
+//THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+//AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+//SOFTWARE.
+//------------------------------------------------------------------------------ -
+
+using Microsoft.MixedReality.Toolkit;
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+
+    public class HandJointFollow : MonoBehaviour
```

Comment:
Do we need / use these scripts in the Debug folder? This one also seems like duplicate functionality to the Follow solver, which allows you to select a specific joint to follow:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/8a2b2f56e649244bad69f20e6f59fceca6782fbe/Assets/MRTK/SDK/Features/Utilities/Solvers/SolverHandler.cs#L64-L66

Reply:
I didn't know how those debug scripts related to our current suite of scripts. I don't think we need them just didn't want to exclude anything that might have been useful by default

Reply:
It might be worth putting these in a .Debug namespace (or examples as mentioned above).

This definitely does have a lot of overlap with our existing solvers - if there is a key difference it would be interesting to call out.

For some of these things where we're not sure, we may can always stuff them in an experimental namespace to give clarity to consumers that this will likely change as we stabilize the area.